__init__.py

```diff
@@ -80,166 +105,40 @@ def _wiki_page_summary(self, result, auto_suggest):
             auto_suggest (bool): True if auto suggest was used to get this
                                  result.
         """
-        lines = 2
+        lines = 20
         summary = wiki.summary(result, lines, auto_suggest=auto_suggest)
 
-        if "==" in summary or len(summary) > 250:
-            # We hit the end of the article summary or hit a really long
-            # one.  Reduce to first line.
+        if "==" in summary or len(summary) > 2500:
+            # We hit the end of the article summary or hit a really 
+            # long one.  Reduce to first line.
+            # TODO - this could be better, correct this eventually
```

Comment:
I assume there has been some discussion behind this change to make the first reply much longer so feel free to disregard this comment.

To me it doesn't quite work for the voice assistant use-case. I preferred the short answer and follow up if needed. I generally don't want a first reply longer than 30 seconds maybe. If I want all the details I'd read the article (or as it is now, ask for more info)

Reply:
i agree with @forslund this change means that in most interactions with wikipedia skill i need to interrupt mycroft and ask it to stop

in my wikipedia skill i use "tell me more" intents via adapt context and read one sentence at a time. Often i end up reading the summary in the GUI and not asking for more info

Reply:
Ah, understood guys. The concept here is barge-in now works so saying 'stop' is less friction then saying 'more' as it is more intuitive (not everybody will figure out the word 'more' does something but they will figure out quickly that 'stop' stops playback). I have considered adding a config value - maybe name it 'auto_more' that would retain existing functionality as I am always concerned with breaking changes. Do you think this would be better? If I do add it I am assuming it would apply to all common query skills (ddg also for example).

Reply:
Understood, personally I think that is a mistake since I will more oftenly be forced to say "stop" than I am now need to say "more". But as I said that's my usecase for the skill and may not reflect the majoritys use-case.

I can hack it into working the way I like to use it

Reply:
I will make it a config param so you can turn manual 'more' on/off.